data/human/campaign events.txt

```diff
@@ -155,11 +155,89 @@ mission "event: war begins"
+			choice
+				`	"I'll get out of the way, then."`
+					goto end
+				`	"Sorry, I didn't know. I'll be out of your hair as soon as I can lift off."`
+					goto end
+				`	"Is there any way I can help?"`
+					goto help
+
+			label martini
+			action
+				set "bombing witness"
+
+			`	As you descend towards the surface of Martini, you are met by an uncharacteristic layer of thick cloud, almost smoky in appearance. The clouds completely obscure Martini's main spaceport, as well as the rest of the capital city. Perhaps because of this, the landing controller instructs you to make your way to an auxiliary spaceport about ten kilometers from the city limits.`
+			`	The moment you step off your ship you realize something is very wrong. Heavily armed Navy personnel flank either side of the spaceport building's exit, and more soldiers are patrolling the port. The main road to the city echoes with sirens, and a liaison officer is herding a group of civilians toward a military transport parked precariously in a nearby field. Still more people are stretched out on cots in a makeshift field hospital in the center of the spaceport.`
+			`	As soon as the last mobile civilian is safely aboard the transport, the liaison officer turns to face you. "I'm sorry, Captain, but I'm going to have to ask you to leave the planet. We've got another transport fleet coming in, and we need all the pads free."`
+			choice
+				`	"Of course."`
+					goto summary
+				`	"What's happening?"`
+				`	"What makes you think you can just order me to leave?"`
+					goto noncompliance
+
+			`	The liaison officer sighs. "Captain, I'm afraid the planet is in a state of emergency. Yesterday, some rogue planets down south decided they wanted to secede from the Republic. Sent Parliament an ultimatum, wanted electoral and tax reform, free trade, all the same things they've been complaining about for years. No big deal. Three hours ago, though, someone - quite possibly this same 'Free Worlds' group - dropped nuclear bombs the Navy Yard on Geminus and on the Galactic Stock Exchange here on Martini. At least, we presume that was the target, since the bomb took about a quarter of the city with it, as well as the main spaceport and half of the planet's medical resources. We've got about a hundred thousand injured people, another four hundred thousand to screen and treat for radiation poisoning, and another million people to evacuate."`
```

Comment:
There's a few things about Martini that make me think it should be low (perfectly terraformed rich tourist world = lots of people living in the country, Martini having multiple "main cities", etc.), but you're probably right that this estimate is much too low.

Revision:

The semi-official population of Martini is about 300 million (source [Derpy's Human Population Sheet](https://docs.google.com/spreadsheets/d/1dWKimAiiXrlVCibdjyiFNqtQJdiXJqB7F0Cs9La0Z60/edit?gid=0#gid=0)), similar to the US. So, I'm going to say that the main city is comparable in size to New York.

I used [Nukemap](https://nuclearsecrecy.com/nukemap/?&kt=800&lat=40.72422&lng=-73.9961&airburst=0&hob_ft=0&casualties=1&fallout=1&ff=10&fallout_angle=157&psi=3000,200,20,5,1&rem=100,500&zm=9) to simulate a nuclear bomb drop (surface burst) on NYC. 
NYC allegedly has an area of 1200 km^2, so I ramped up the yield on the warhead until the circle for 100% chance of 3rd degree burns hit 300km^2, at 800 kilotons.

I'm presuming that because the Syndicate wanted to start a war and not to destroy one of their richest customer planets entirely, they would use a relatively clean nuke, so I dropped the fission level on the simulator to 10%, which seems to minimize fallout.

That gave me 1.17 million fatalities and 1.41 million injuries, pre-fallout.

So, my original estimates were definitely way too low.
Adjusted estimate:
1 million dead
1.5 million injured
might as well not talk about radiation screening
maybe 5 million people to evacuate?